### PR TITLE
NikitK/ Fix flash of unstyled text (FOUT) on payment method page

### PR DESCRIPTION
**Pull request by NikitK-deriv**
Changes:

-    flash of footer on reload page. rename of footer usage

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/layout/layout.tsx [hunk 1]
```diff
@@ -255,39 +256,43 @@ const Layout = ({
...
             Navigation = <NavCareers is_besquare />
             FooterNav = <BeSquareFooter />
             break
         case 'careers':
             Navigation = <NavCareers />
-            FooterNav = <Footer no_language={true} type={type} />
+            FooterNav = <LoadableFooter no_language={true} type={type} />
             break
         case 'transparent':
+            Navigation = <NavTransparent is_ppc_redirect={is_ppc_redirect} is_ppc={is_ppc} />
```
Comment (mitra-deriv) [inline src/components/layout/layout.tsx:286]: Hi @NikitK-deriv I'm just curious to know what is the `is_ppc` exactly? 🙏 

Also may I know how we are fixing the issue exactly I couldn’t understand it, many thanks in advance 🙏

Comment (NikitK-deriv) [reply src/components/layout/layout.tsx:286]: Yes, sure @mitra-deriv 

Seems like `is_ppc` property related to landing pages. For some special cases. I'll check landing for payment methods and maybe have sense to remove that. Good catch 👍 

So main issue was that footer component didn't exist after reload page due that footer for payment methods used `Loadable` function to load component later. That's why footer flash every time
Common behaviour of react is show whole page when all components already rendered. I decided to import component without loadable function and save common react behavior for this specific page

Hope it's clear 🙇

Comment (habib-deriv) [reply src/components/layout/layout.tsx:286]: Hi @NikitK-deriv , as the issue is for Payment Method page, can you implent this only for that page? If needed we can implement for all pages for a different card

Comment (NikitK-deriv) [reply src/components/layout/layout.tsx:286]: Hey @habib-deriv it's exactly implemented only for payment agent page. The other didn't change

**Review by habib-deriv — APPROVED** [review]
LGTM

**Review by habib-deriv — DISMISSED** [review]
LGTM

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**